server/server.js

```diff
@@ -370,6 +380,24 @@ app.get(`${apiBaseUrl}/employer-sites`,
     return res.json({ data: result });
   }));
 
+app.get(`${apiBaseUrl}/employer-sites/:id`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    const user = req.hcapUserInfo;
+    const [result] = await getSiteByID(req.params.id);
+    logger.info({
+      action: 'employer-sites-detail_get',
+      performed_by: {
+        username: user.username,
+        id: user.id,
+      },
+      eeoi_id: req.query.id,
+      siteID: result.siteID,
+    });
+    return res.json(result);
+  }));
+
 // In development
```

Comment:
Tiny nitpick but if we're in this file anyway can we update this comment? :)

Reply:
Also is this endpoint deprecated now?

Reply:
Okay yes I think this is deprecated.

My vote - leave it in the codebase for now and document this in the comment here. I know Java has an @deprecated tag, not sure about JS. We can state here which endpoint replaces it and mention that legacy logs will need to be search according to the logging shown below

Thoughts?

Reply:
Quick google says that JSDoc also has a deprecated flag.

Reply:
Awesome 🙌